packages/xo-server/src/xo-mixins/authentication.js

```diff
@@ -113,7 +113,7 @@ export default class {
 
         if (typeof result === 'string') {
```

Comment:
This case does not appear to be still in use, I think we should simply remove it.